client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
It is advisable to include testing libraries in the devDependencies section, as failing to do so may result in their being bundled into the React application, consequently leading to an increase in the overall bundle size.

Reply:
I don't know what you are talking about. Maybe I have not studied it yet.

Reply:
Have shared a link to know more about the difference between dependencies and devDependencies. Please go through it.